content/scalability/docs/control-plane.md

```diff
@@ -0,0 +1,164 @@
+# Kubernetes Control Plane
+
+The Kubernetes control plane is the Kubernetes components such as the API Server, kubelet, and Kubernetes Controller Manager. Scalability limits of these components are different depending on what you’re running in the cluster, but the areas with the biggest impact to scale include the Kubernetes version, utilization, and individual Node scaling.
+
+## Use EKS 1.24 or above
+
+EKS 1.24 introduced a number of changes and switches the default container runtime to [containerd](https://containerd.io/) instead of docker. Containerd helps clusters scale by increasing individual node performance by limiting container runtime features to closely align with Kubernetes’ needs. Containerd is available in every supported version of EKS and if you would like to switch to containerd in versions prior to 1.24 please read the documentation about [Dockershim deprecation](https://docs.aws.amazon.com/eks/latest/userguide/dockershim-deprecation.html).
+
+## Automate Amazon Machine Image (AMI) updates
```

Comment:
How is automation of AMI updates helping with scalability of Kubernetes control plane? It will help to the reader if we can call that out in couple of lines.